app/models/student.rb

```diff
@@ -0,0 +1,13 @@
+class Student < ApplicationRecord
+  validates :first_name, presence: true
```

Comment:
You can combine some of these lines...

    validates :first_name, :last_name, :cell_phone, :home_phone, :work_phone, :email, presence: true